### PR TITLE
Added user sync support into adapter

### PR DESCRIPTION
**Pull request by lemmadev**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change

We have added user sync support into our adapter. 

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/lemmaBidAdapter.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
-import * as utils from '../src/utils.js';
+import * as utils from '../src/utils';
```
Comment (robertrmartinez) [inline modules/lemmaBidAdapter.js:1]: should include .js extension.

**Review by robertrmartinez — CHANGES_REQUESTED** [review]
please add associated unit tests

File: modules/lemmaBidAdapter.js (hunk 1 above)
Comment (lemmadev) [reply modules/lemmaBidAdapter.js]: updated file extension.

**Review by FilipStamenkovic — CHANGES_REQUESTED** [review]
There are couple of small errors that should be fixed before merging.  

You can just run `gulp lint` and take a look at the output, it will fix `'` vs `"` format and it will let you know about other errors.

File: modules/lemmaBidAdapter.js [hunk 2]
```diff
@@ -5,10 +5,13 @@ import { BANNER, VIDEO } from '../src/mediaTypes.js';
 var BIDDER_CODE = 'lemma';
 var LOG_WARN_PREFIX = 'LEMMA: ';
 var ENDPOINT = 'https://ads.lemmatechnologies.com/lemma/servad';
+var USER_SYNC = "https://sync.lemmatechnologies.com/js/usersync.html?"
```
[the remark this answers is not in the document]
Comment (lemmadev) [reply modules/lemmaBidAdapter.js]: Fixed format error.

File: modules/lemmaBidAdapter.js [hunk 3]
```diff
@@ -57,6 +60,29 @@ export var spec = {
   interpretResponse: (response, request) => {
     return parseRTBResponse(request, response.body);
   },
+  getUserSyncs: (syncOptions, responses, gdprConsent, uspConsent) => {
+    let syncurl = USER_SYNC + "pid=" + pubId;
```
[the remark this answers is not in the document]
Comment (lemmadev) [reply modules/lemmaBidAdapter.js]: Fixed format error.

File: test/spec/modules/lemmaBidAdapter_spec.js [hunk 4]
```diff
@@ -331,5 +331,39 @@ describe('lemmaBidAdapter', function() {
...
+        expect(spec.getUserSyncs({ iframeEnabled: true }, {}, undefined, '1NYN')).to.deep.equal([{
+          type: 'iframe', url: `${syncurl_iframe}&us_privacy=1NYN`
+        }]);
+      });
+
+      it('GDPR', function() {
+        expect(spec.getUserSyncs({ iframeEnabled: true }, {}, { gdprApplies: true, consentString: 'foo' }, undefined)).to.deep.equal([{
+          type: 'iframe', url: `${syncurl_iframe}&gdpr=1&gdpr_consent=foo`
+        }]);
+        expect(spec.getUserSyncs({ iframeEnabled: truev}, {}, { gdprApplies: false, consentString: 'foo' }, undefined)).to.deep.equal([{
```
[the remark this answers is not in the document]
Comment (lemmadev) [reply test/spec/modules/lemmaBidAdapter_spec.js]: Fixed format error and typo error

**Review by FilipStamenkovic — APPROVED** [review]
LGTM

**Pull request merged**